2021_12_14_001023_alter_imobils_table_remove_column.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AlterImobilsTableRemoveColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('imobils', function (Blueprint $table) {
+            if (Schema::hasColumn('imobils','description')) {
+                $table->dropColumn('description');
+            }
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('imobils', function (Blueprint $table) {
+            $table->string('description')->after('balcony');
```

Comment:
description was not a string type

Reply:
change string into text    Schema::table('imobils', function (Blueprint $table) {
                                                     $table->text('description')->after('balcony');
                                          });